docs/contributing/featureflags.md

```diff
@@ -27,50 +27,51 @@ where the flags are sourced.
 The source of the flags is dependent upon the Bitwarden server instance that is being used, as for
 client development the flags are served from the Bitwarden API.
 
-| Server configuration | Flag source                                            |
-| -------------------- | ------------------------------------------------------ |
-| Local development    | Local `flags.json` file                                |
-| Self-hosted          | Flags are "off", unless local `flags.json` is provided |
-| QA Cloud             | LaunchDarkly QA                                        |
-| Production Cloud     | LaunchDarkly Production                                |
+| Server configuration | Flag source                                    |
+| -------------------- | ---------------------------------------------- |
+| Local development    | Local JSON file                                |
+| Self-hosted          | Flags are "off", unless local JSON is provided |
+| QA Cloud             | LaunchDarkly QA                                |
+| Production Cloud     | LaunchDarkly Production                        |
 
 :::warning Self-hosted support
```

Comment:
💭 I never know which of these is best. I saw a "caution" on another PR and wonder if that is a better fit now.

Reply:
I like that!  I've updated it.  I didn't realize caution was an option.